src/components/quiz/UserDetailsForm.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from "react";
+
+interface User {
+  name: string | null;
+  displayPic: string | null;
+}
+
+export const UserDetailsForm: React.FunctionComponent = ({status,setStatus}: any) => {
+
+  const [userDetails, setUserDetails] = useState<User>({
+    name: null,
+    displayPic:null,
+  });
+  
+
+  function onSubmit(event: any) {    
+    event.preventDefault();     
+    setStatus(true);   
+  }
+
+  return (
+    <div>
+      <h2>Enter your details cadet !</h2>
+   
+      <form action="post">
+        <div className="name">
+          <label> Name: </label>
+          <input
+            name="Name"
+            type="text"
+            onChange={(e) =>
+              setUserDetails({ ...userDetails, name: e.target.value })
+            }
+          />
```

Comment:
Labels and inputs must either:
1. Be linked together by their ID:
```suggestion
          <label htmlFor="user-details-form-name"> Name: </label>
          <input
            id="user-details-form-name"
            name="Name"
            type="text"
            onChange={(e) =>
              setUserDetails({ ...userDetails, name: e.target.value })
            }
          />
```
2. Have the input inside the label:
```suggestion
          <label>
            Name:
            <input
              name="Name"
              type="text"
              onChange={(e) =>
                setUserDetails({ ...userDetails, name: e.target.value })
              }
            />
          </label>
```
to link them together!